src/storm/utility/solver.cpp

```diff
@@ -1,14 +1,14 @@
 #include "storm/utility/solver.h"
 
-#include "storm/solver/GlpkLpSolver.h"
-#include "storm/solver/GurobiLpSolver.h"
-#include "storm/solver/SoplexLpSolver.h"
-#include "storm/solver/Z3LpSolver.h"
-
+#include "storm/exceptions/InvalidOperationException.h"
```

Comment:
Why this? I can not find any usage of `InvalidOperationException`.

Reply:
We are using it here: https://github.com/moves-rwth/storm/blob/master/src/storm/utility/solver.cpp#L134

Reply:
Sorry, cmd+f apparently didn't work somehow... Thanks!